tests/test_config_registry_token.py

```
@@ -42,3 +42,13 @@ def test_clear_returns_none(data_dir):
 def test_set_empty_string_raises(data_dir):
     with pytest.raises(ValueError):
         config.set_registry_token("", data_dir=data_dir)
+
+
+def test_positional_path_raises_typeerror(data_dir):
+    with pytest.raises(TypeError):
+        config.set_registry_token("tok", "/tmp/probe-datadir")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/probe-datadir"

(S108)